docs/source/conf.py

```diff
@@ -51,53 +47,75 @@
+breathe_default_project = project
+
+# Tell sphinx not to process the doxygen output directory when searching for
+# files to render into html.
+exclude_patterns = ['_doxygen/**']
 
 # -- Exhale setup ------------------------------------------------------------
 exhale_args = {
-        "containmentFolder": "./api",
-        "rootFileName":      "library_root.rst",
-        "rootFileTitle":     "Code",
-        "doxygenStripFromPath": "../../BattleNetwork",
-        "createTreeView":    True,
-        "exhaleExecutesDoxygen": True,
-        # "exhaleDoxygenStdin": "INPUT = ../../BattleNetwork",
-        "exhaleUseDoxyfile": True,
-        "unabridgedOrphanKinds": {"dir", "file" }
-        }
+    "containmentFolder": "./api",
+    "rootFileName": "library_root.rst",
+    "rootFileTitle": "Code",
+    # NOTE: see STRIP_FROM_PATH in Doxyfile.  If that updates, this must too.
+    "doxygenStripFromPath": "../../BattleNetwork",
+    "fullToctreeMaxDepth": 1,
+    # See edits in source_read.  This tricks Exhale into not including a
+    # top-level "Full API" heading on the unabridged document.
+    "fullApiSubSectionTitle": "",
+    "createTreeView": False,  # See source_read, they get pruned.
+    "exhaleExecutesDoxygen": True,
+    "exhaleUseDoxyfile": True,
+    "unabridgedOrphanKinds": {"dir"}
+}
 
 primary_domain = 'cpp'
-
 highlight_language = 'cpp'
 
-# on_rtd is whether we are on readthedocs.org, this line of code grabbed from docs.readthedocs.org
-on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
-
-html_favicon = 'favicon.gif'
-html_logo    = 'favicon.gif'
-html_theme   = 'bootstrap'
-html_theme_options = {
-        'navbar_site_name': "Open MMBN",
-        'navbar_links': [
-            ("Contributing", "https://github.com/TheMaverickProgrammer/battlenetwork/wiki/Contributing", True)
-            ],
 
-        'source_link_position': "exclude",
+def source_read(app, docname, source):
+    # NOTE: these patches should work until exhale 1.x is released.
 
-        'navbar_class': "navbar",
+    # source is a length one list with it's single element being the source
+    # of the file being processed.  Modify source[0] to change what Sphinx
+    # will render.
 
-        'navbar_fixed_top': "true",
+    # Delete both view hierarchies since this project has a mostly flat
+    # structure.  Rely on the unabridged API instead.
+    if docname == "api/library_root":
+        source[0] = re.sub(
+            r"\.\. include:: .*_view_hierarchy.rst", "", source[0]
+        )
 
-        'bootswatch_theme': "sandstone",
+    # Orphan the view hierarchy documents since they aren't used anymore.
+    if docname in {"api/class_view_hierarchy", "api/file_view_hierarchy"}:
+        source[0] = ":orphan:\n\n" + source[0]
 
-        'bootstrap_version': "3",
-        }
 
-if not on_rtd:
-    import sphinx_bootstrap_theme
-    html_theme = 'bootstrap'
-    html_theme_path = [sphinx_bootstrap_theme.get_html_theme_path()]
```

Comment:
in sphinx 2.x you just need to do `html_theme = "<name>"`, `html_theme_path` not needing to be set anymore.